### PR TITLE
Setup induction model wrapper

### PR DESCRIPTION
**Pull request by semibran**
Results are inconclusive, not quite ready for submission

TODO:
- Determine optimal location for override predicates
- Find more effective predicate clauses from dataset
- Use SVM to extract more complex feature relationships(?)

### CODE REVIEW

File: main_induction.py [hunk 1]
```diff
@@ -123,13 +134,13 @@ def sorted_models(model_scores):
 
 
     model_scores = {}
-    kfold = KFold(n_splits=NUM_K_FOLD_SPLITS)
+    kfold = StratifiedKFold(n_splits=NUM_K_FOLD_SPLITS)
```
Comment (semibran) [inline main_induction.py:137]: this isn't super important but apparently it helps balance out the label classes per-fold

File: main_induction.py [hunk 2]
```diff
@@ -51,13 +46,24 @@ class ModelPerformance:
...
 
+def _should_accept_claim(claim):
+    return (claim['feature11'] == 5
+        or claim['feature9'] == 0
+        or claim['feature13'] == 4
+        or claim['feature14'] == 3
+        or claim['feature18'] == 1)
+
+def _should_reject_claim(claim):
+    return claim['feature7'] == 3
```
Comment (semibran) [inline main_induction.py:65]: :brain:

**Review by jonathan-paugh — APPROVED** [review]
Looks good, would like to mess around with the predicates to see if I can figure anything out.

**Comment by semibran** [comment]
yup it could help for testing to remove most of the modifiers/turn down number of k folds but might want to be careful with the acceptance predicate cause not all feature that look correlated in the classification graphs actually boost the results, sometimes it'll just start accepting a bunch of nonsense claims and drive precision way down

**Pull request merged**